### ISSUE TITLE
lost the authentication code

### ISSUE THREAD

**Issue by edgalraphael** [mattrubin/authenticator#240]
I have created an account at homeblockcoin.com. id is edgalraphael and email is [redacted]. I set up an authentication code to protect my account but I lost my phone so I could not go to Id edgalraphael. hope you can help me remove this authentication code. Thank you very much

**Comment by mattrubin** [comment]
The authenticator app stores authentication tokens, but isn't associated with any website and doesn't have control over any of your accounts. If you have an encrypted backup of your lost phone, you may be able to restore it to a new phone to recover your authenticator tokens. To reset two-factor authentication on your HomeBlockCoin account, you will need to contact the administrators of [homeblockcoin.com](https://homeblockcoin.com).

**Comment by edgalraphael** [comment]
I took a screenshot but unfortunately my phone was lost and I could not get the capture image encoded. I tried to contact the administrator of homeblockcoin.com but did not receive a response from them. Please help me remove the authentication code, plz!

**Comment by edgalraphael** [comment]
Tôi chụp ảnh màn hình nhưng tiếc là điện thoại của tôi đã bị mất và tôi
không thể chụp ảnh được mã hóa. Tôi đã cố liên hệ với quản trị viên của
homeblockcoin.com nhưng đã không nhận được phản hồi từ họ. Xin vui lòng
giúp tôi gỡ bỏ mã xác thực, plz!

2017-11-27 2:45 GMT+07:00 Matt Rubin <notifications@github.com>:

**Comment by beaucollins** [comment]
I just gave the homeblockcoin.com website a run through. Unfortunately during the two-factor setup process they don't even provide back up codes to the user. There is also zero documentation from their website on how to get two-factor removed from an account when you have lost access. 

@edgalraphael you will need to get a hold of someone at homeblockcoin.com to help you regain access to your account.

Another thing worth noting is that the QR code image I was given did not work and I had to manually enter the secret on my device.
